anchor-service/src/high_water_mark_store.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::Result;
+use ceramic_sql::sqlite::SqlitePool;
+
+#[derive(Debug)]
+pub struct HighWaterMarkStore {
```

Comment:
I'm fine with this name since that's all it does right now, but wondering if we could call it Anchor*Store (AnchorRequestStore, AnchorServiceStore, AnchorStore)?